modules/claims_api/spec/fixtures/form_2122_json_api.json

```diff
@@ -2,9 +2,11 @@
   "data": {
     "type": "form/2122",
     "attributes": {
-      "poaCode": "074",
-      "poaFirstName": "John",
-      "poaLastName": "Doe"
+      "serviceOrganization": {
+        "poaCode": "074",
```

Comment:
We need to confirm that what actually gets serialized and sent to BGS hasn't changed, I knew that nesting the poaCode would trigger this change I'm just not sure if that's already in the feature branch or if you're doing it in this PR?